lzy/allocator/src/main/java/ai/lzy/allocator/services/AllocatorService.java

```diff
@@ -119,6 +119,18 @@ public void createSession(CreateSessionRequest request, StreamObserver<LongRunni
 
         final var operationId = idGenerator.generate("create-session-");
         final var sessionId = idGenerator.generate("sid-");
+        try {
+            allocationContext.networkPolicyManager().createNetworkPolicy(
+                sessionId,
+                request.getNetPolicyRulesList().stream().map(this::toPolicyRule).toList()
+            );
+        } catch (Exception ex) {
+            allocationContext.metrics().createSessionError.inc();
+
+            LOG.error("Cannot create session: {}", ex.getMessage(), ex);
+            responseObserver.onError(Status.FAILED_PRECONDITION.withDescription(ex.getMessage()).asException());
+            return;
+        }
```

Comment:
Maybe we need new action here?
I think creating new netPolicy can be long request